notebooks/financial-docs-10k-example.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "##### In this example, we will use request library to make API call to Aryn Partitioning Service. We'll send a financial document (a 10k) to the service and then extract a table from the JSON object returned. We'll turn that table into a pandas dataframe and then perform some analysis on it.\n",
+    "\n",
+    "##### To access the Aryn Partitioning Service for fast, GPU-powered performance go to [aryn.ai/sign-up ](aryn.ai/sign-up) and sign up to get a free API key for the service. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Run pip install aryn-sdk first\n",
+    "import aryn_sdk\n",
+    "import boto3\n",
+    "import pandas as pd\n",
+    "import os\n",
+    "from io import BytesIO\n",
+    "from aryn_sdk.partition import partition_file, tables_to_pandas"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# visit https://www.aryn.ai/cloud to get a key\n",
+    "aryn_api_key = os.environ['ARYN_API_KEY'] "
```

Comment:
I think it's standard to have this as `"YOUR_ARYN_API_KEY"`, would be worth rechecking this

Reply:
I think the assumption here is the customer would already have setup the environment variable as  "ARYN_API_KEY" and not as "YOUR_ARYN_API_KEY". I see the point you're making, but I think most customers won't be setting the env variable as "YOUR_ARYN_API_KEY".

Reply:
Yes, but take a look at the other notebooks. It's been standardized differently

Reply:
Oh i see, other notebooks are not pulling it from the environment and just setting it as "YOUR_ARYN_API_KEY" and expecting the customer to set it. I can discuss this with you offline.